packages/block-library/src/embed/core-embeds.js

```diff
@@ -21,255 +21,257 @@ import {
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { createBlock } from '@wordpress/blocks';
+import {
+	__
+} from '@wordpress/i18n';
+import {
+	createBlock
+} from '@wordpress/blocks';
 
-export const common = [
-	{
+export const common = [{
 		name: 'core-embed/twitter',
 		settings: {
 			title: 'Twitter',
 			icon: embedTwitterIcon,
-			keywords: [ 'tweet' ],
```

Comment:
Please don't delete the Space, see the JS Coding Standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

Reply:
Oops, must have been the beautify on save plugin I'm using. Sorry about that, will fix.